packages/selleo-design-core/src/Avatars.tsx

```diff
@@ -0,0 +1,68 @@
+import { h } from 'preact';
+import cx from 'classnames';
+
+export type AvatarProps = {
+  hideName?: boolean;
+  size: 'regular' | 'big';
+  image?: h.JSX.Element | null;
+  username?: string;
+  hasCustomImageSize?: boolean;
+};
+
+export function Avatar({
+  hideName = false,
+  size,
+  image,
+  username,
+  hasCustomImageSize,
+}: AvatarProps) {
+  const imageClasses = cx(
+    `rounded-full overflow-hidden bg-white flex justify-center items-center`,
+    {
+      'mr-1': !hideName,
+      'w-[24px] h-[24px]': size === 'regular',
+      'w-[32px] h-[32px]': size === 'big',
+      'w-[64px] h-[64px] mr-2': hasCustomImageSize && size === 'regular',
+      'w-[111px] h-[111px] mr-4': hasCustomImageSize && size === 'big',
```

Comment:
as we discussed Avatars should not have Testimonial logic inside them and also hasCustomSize is an unnecessary prop

Reply:
Yes, already done :)